queries/models/dashboard_subsidio_sppo/schema.yml

```
@@ -701,4 +701,6 @@ models:
               name: not_null__indicador_viagem_dentro_limite__viagens_remuneradas
               where: "data between date('{date_range_start}') and date('{date_range_end}')"
       - name: indicador_penalidade_judicial
-        description: "{{ doc('indicador_penalidade_judicial') }}"
\ No newline at end of file
+        description: "{{ doc('indicador_penalidade_judicial') }}"
+      - name: id_execucao_dbt
+        description: "{{ doc('id_execucao_dbt') }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)